src/components/BreweryCard/BreweryCard.tsx

```diff
@@ -0,0 +1,16 @@
+import "./BreweryCard.css"
+
+export default function BrieweryInterface({brewery}:{brewery:any}) {
```

Comment:
rename BrieweryInterface to BreweryCard - try to add the type of the brewery. Types should be exported in a separate folder and then imported wherever you want them